bitmark-references/security.markdown

```diff
@@ -0,0 +1,73 @@
+# Security feature of the Bitmark Blockchain
+
```

Comment:
This is another article where we really need a good intro that says what the article is about.

Something like:

"The Bitmark Property System uses signatures and hashes to secure its transactions and blocks."